nameable.rb

```diff
@@ -0,0 +1,11 @@
+# class nameable
+
+class Nameable
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ]👋 Hello! Great job on the code implementation! 😊 I've noticed a small issue that's causing an error when running the app:

💡 **Issue:** 
The `Nameable` class currently expects an argument to be passed to its constructor, which doesn't align with the project requirements.

🛠️ **Suggestion:**
To match the requirements, you can update the `Nameable` class like this:

```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end
```
This change can provide you with:
🔍 Clarity in error messages.
🚀 A simpler codebase without unnecessary constructors.
📜 A common and recognized programming convention.
👍 Encouragement for creating subclasses.

It can enhance the maintainability of your code. Give it a try! 😊

Reply:
- Well resolved 👍🏽